lib/storage/src/content_manager/toc/mod.rs

```diff
@@ -782,84 +585,6 @@ impl TableOfContent {
         Ok(aliases)
     }
 
-    /// Paginate over all stored points with given filtering conditions
-    ///
-    /// # Arguments
-    ///
-    /// * `collection_name` - which collection to use
-    /// * `request` - [`ScrollRequest`]
-    /// * `shard_selection` - which local shard to use
-    ///
-    /// # Result
-    ///
-    /// List of points with specified information included
-    pub async fn scroll(
-        &self,
-        collection_name: &str,
-        request: ScrollRequest,
-        read_consistency: Option<ReadConsistency>,
-        shard_selection: Option<ShardId>,
-    ) -> Result<ScrollResult, StorageError> {
-        let collection = self.get_collection(collection_name).await?;
-        collection
-            .scroll_by(request, read_consistency, shard_selection)
-            .await
-            .map_err(|err| err.into())
-    }
-
-    pub async fn update(
-        &self,
-        collection_name: &str,
-        operation: CollectionUpdateOperations,
-        shard_selection: Option<ShardId>,
-        wait: bool,
-        ordering: WriteOrdering,
-    ) -> Result<UpdateResult, StorageError> {
-        let collection = self.get_collection(collection_name).await?;
-
-        // Ordered operation flow:
-        //
-        // ┌───────────────────┐
-        // │ User              │
-        // └┬──────────────────┘
-        //  │ Shard: None
-        //  │ Ordering: Strong
-        // ┌▼──────────────────┐
-        // │ First Node        │ <- update_from_client
-        // └┬──────────────────┘
-        //  │ Shard: Some(N)
-        //  │ Ordering: Strong
-        // ┌▼──────────────────┐
-        // │ Leader node       │ <- update_from_peer
-        // └┬──────────────────┘
-        //  │ Shard: Some(N)
-        //  │ Ordering: None(Weak)
-        // ┌▼──────────────────┐
-        // │ Updating node     │ <- update_from_peer
-        // └───────────────────┘
```

Comment:
I'm missing this comment after refactoring (7c5f00a70307a0c9ee646f107ccfffb9c05b0b66). I think it'll be useful to keep.

Reply:
Nice catch. I'll add it back now.

Reply:
Done.